jit-compiler/tests/execution.rs

```diff
@@ -0,0 +1,39 @@
+use test_log::test;
+
+use powdr_number::GoldilocksField;
+use powdr_pil_analyzer::analyze_string;
+
+fn compile(input: &str, symbol: &str) -> fn(u64) -> u64 {
+    let analyzed = analyze_string::<GoldilocksField>(input).unwrap();
+    powdr_jit_compiler::compile(&analyzed, &[symbol]).unwrap()[symbol]
+}
+
+#[test]
+fn identity_function() {
+    let f = compile("let c: int -> int = |i| i;", "c");
+
+    assert_eq!(f(10), 10);
+}
+
```

Comment:
I'm not sure if this is the right place, but I would like some test to fail until more types are supported.

```suggestion
#[test]
#[should_panic = "Only (int -> int) functions and columns are supported, but requested  c: int -> bool"]
fn invalid_function() {
    let _ = compile("let c: int -> bool = |i| true;", "c");
}
```